src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIAnnouncementsService.java

```diff
@@ -97,23 +93,22 @@ public void deleteAnnouncement(@RequestBody AnnouncementRequest announcementRequ
 	/**
 	 * Get user read status for current announcement
 	 * @param announcementID {@link Long} identifier for the {@link Announcement}
-	 * @param params {@link DataTablesParams} parameters for current DataTable
-	 * @return {@link DataTablesResponse} containing the list of users.
+	 * @param tableRequest details about the current page of the table requested
+	 * @return a {@link TableResponse} containing the list of users.
 	 */
 	public @ResponseBody
```

Comment:
Can you remove this `@ResponseBody` annotation, that is only used on controllers, not services.  Actually, looking through this file, there is a bunch of annotations that should not be there.  The only one that should is the `@Component`.  Can you please remove all the `@RequestBody` and `@ResponseBody` annotations?

Reply:
Done. Please see 2024268b62109feef1af7dd431a0ebf4f5cb0143.